src/pages/admin/AdminUsersPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { listInstitutionUsers } from '../../api/userApi';
+import Heading from '../../components/Heading';
+import SubHeading from '../../components/SubHeading';
+import { useSelector } from 'react-redux';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import { UserAdmin, RoleName } from '../../types/user.types';
+import styled from 'styled-components';
+import Card from '../../components/Card';
+import { Button, FormControlLabel, Checkbox } from '@material-ui/core';
+import UserList from './UserList';
+import NormalText from './../../components/NormalText';
+
+const StyledButton = styled(Button)`
+  margin-top: 1rem;
+`;
+
+const StyledContainer = styled.div`
+  margin-bottom: 2rem;
+`;
+
+const AdminUsersPage: FC = () => {
+  const { t } = useTranslation('profile');
+  const user = useSelector((store: RootStore) => store.user);
+  const [userList, setUserList] = useState<UserAdmin[]>([]);
+  const [autoAssignCreators, setAutoAssignCreators] = useState(true);
+
+  useEffect(() => {
+    const getUsers = async () => {
+      const users = await listInstitutionUsers(user.institution);
+      // TODO backend
+      // if (users?.error) {
+      //   dispatch(addNotification(t('feedback:error.get_publications'), 'error'));
+      // } else {
```

Comment:
ser ingen problemer med å ta med dette selv om backend ikke er klar. kan jo prøve å hardkode error-respons selv også, for å se at det virker

Reply:
vi har vel gjort det litt forskjellig der vi ikke har backend.  Trenger nok en gjennomgang snart

Reply:
Velger å ikke endre det i denne pr'en